### DISCUSSION TITLE
Not able to follow an account: what am I doing wrong?

### DISCUSSION THREAD

**Discussion by aborruso** [mastodon/mastodon#20263] [Q&A]
Hi,
I'm not able to follow an account (this [one](https://respublicae.eu/@EU_opendata)).

These are my steps: https://www.youtube.com/watch?v=UhTI7hLxjnc

What am I doing wrong?

Thank you

**Comment by MrGeneration** [comment]
I don't speak italian I'm afraid. I can follow said account without issues.
Maybe you're blocked or it's an temporary issue?

What you can do is to search for the account in your local instance (`@EU_opendata@respublicae.eu`) and try to follow it from there.
If the issue persists it is either an account or server issue.

![screencast](https://user-images.githubusercontent.com/6549061/201132673-0f5b4cd9-a2df-45fd-843e-18c0203042ba.gif)

**Reply by aborruso** [reply]
Thank you. I have discovered what's the reason: the manager has blocked that account :(

**Reply by MrGeneration** [reply]
Unlucky, sorry to hear. :-/
